src/falcon_cors/__init__.py

```diff
@@ -426,7 +426,8 @@ def _get_requested_headers(self, req):
         if raw_header is None:
             return []
         else:
-            return [requested_header.strip() for requested_header in raw_header.split(',')]
+            return [requested_header.strip() for requested_header in raw_header.split(',')
+                    if requested_header.strip() != '']
```

Comment:
Admittedly a bit wasteful, but it was the smallest change. If desired, I can of course use an alternative approach to prune empty strings from the list after construction.